packages/core/integration-tests/test/javascript.js

```diff
@@ -6326,6 +6327,26 @@ describe('javascript', function () {
     assert.equal(await res.default, 'target');
   });
 
+  it('should retain unicode escape sequences', async function () {
```

Comment:
Is it worth leaving a comment explaining why this would be beneficial? Maybe: _"Not all environment deal correctly with special unicode characters. #8877"_